src/box/txn_limbo.c

```diff
@@ -387,9 +387,8 @@ txn_limbo_checkpoint(const struct txn_limbo *limbo, struct synchro_request *req,
 	req->confirmed_vclock = vclock;
 }
 
-/** Write a request to WAL. */
-static void
-synchro_request_write(const struct synchro_request *req)
+static int
+synchro_request_write_without_error_handling(const struct synchro_request *req)
```

Comment:
Lets just leave it with the old name? If a function returns an `int`, it usually means it didn't do any final error handling anyway. This is a normal behaviout. And the panic-function we could rename to `synchro_request_write_or_panic()`. How does that sound?

Reply:
`synchro_request_write_xc` means it throws an exception, but it doesn't. We don't have an established naming scheme for panicking functions, so I would go for @Gerold103's version.